mac

```diff
@@ -170,7 +170,7 @@ append_general_dependencies() {
     tap "github/gh"
 
     # mas-cli to install macOS apps
-    brew "mas"
+    tap "mas-cli/tap"
```

Comment:
I do not think there is a need to support older versions of Mac OS. At most, the script should support the current and previous versions. 

![image](https://user-images.githubusercontent.com/696529/158950590-ad7e03a6-99b1-4b0e-befc-201a7b0f776e.png)

Since Catalina is already two versions ago, we can drop support for it.

Reply:
sure, I actually opened this PR seeing the support of Catalina and Mojave in README. But I just realized that the readme was written 2 years ago and the OS Catalina which was "new" then is old now 😄  I'll go ahead and close this one! 😄

Reply:
In that case, do you mind opening another PR to update the README?

Reply:
sure, created one at https://github.com/nimblehq/laptop/pull/30